entries/p/patient.labcorp.com.json

```
@@ -1,14 +1,21 @@
 {
   "Labcorp Patient": {
     "domain": "patient.labcorp.com",
-    "contact": {
-      "form": "https://www.labcorp.com/contact-us/labcorp-patient-portal"
-    },
+    "tfa": [
+      "sms",
+      "totp",
+      "custom-software"
+    ]
+    "custom-software": [
+      "Okta Verify"
+    ],
+    "documentation: "https://patient.labcorp.com/help-center/q-and-a/8000/2"
```

```suggestion
    "documentation": "https://patient.labcorp.com/help-center/q-and-a/8000/2",
```
This should fix the errors you're getting